v2/tools/generator/internal/codegen/pipeline/export_generated_code.go

```diff
@@ -8,17 +8,16 @@ package pipeline
 import (
 	"context"
 	"fmt"
+	"golang.org/x/exp/slices"
+	"golang.org/x/sync/errgroup"
 	"os"
 	"path/filepath"
-	"sort"
 	"sync"
 	"time"
 
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
```

Comment:
minor: I think should be in a separate stanza at the bottom

Reply:
Moved.